lib/game_end_evaluator.rb

```diff
@@ -1,11 +1,11 @@
 class GameEndEvaluator
-  def find_winner(board)
+  def player_won?(board)
     [board.rows, board.columns, board.diagonals].each do |board_state|
```

Comment:
Are you think of something like `board#matrices`, that would return `[board.rows, board.columns, board.diagonals]`? I think this approach is nice because it only creates one point contact between the models. Is this what you had in mind?

I think I like my original approach because it's pretty clear to the reader what is happening.

Reply:
I think that's the general idea! I don't know about matrices but maybe something more descriptive as to their use case so that you get the benefit of your original approach and the cleaner interface.